tests/scalapack/scalapack_04.cc

```diff
@@ -41,38 +41,63 @@ void test(const unsigned int size, const unsigned int block_size, const NumberTy
   ConditionalOStream pcout (std::cout, (this_mpi_process ==0));
 
   // Create SPD matrices of requested size:
-  FullMatrix<NumberType> full_A(size);
+  FullMatrix<NumberType> full_A(size), full_B(size);
```

Comment:
let's create a separate test which checks norms of a general matrix. Better to have more small tests than a couple of big ones. You can name it `scalapack_04_b` if you want to indicate that they are related